MO4/Sniffs/Arrays/ArrayDoubleArrowAlignmentSniff.php

```diff
@@ -75,7 +75,10 @@ public function process(File $phpcsFile, $stackPtr): void
             return;
         }
 
+        // phpcs:disable
```

Comment:
It would be nicer to disable the specific sniff.

Reply:
it's not one, but five:
```
 78 | ERROR | [x] The open comment tag must be the only content on the line (Generic.Commenting.DocComment.ContentAfterOpen)
 78 | ERROR | [ ] Missing short description in doc comment (Generic.Commenting.DocComment.MissingShort)
 78 | ERROR | [x] Block comments must be started with /* (Squiz.Commenting.BlockComment.WrongStart)
 78 | ERROR | [ ] Inline doc block comments are not allowed; use "/* Comment */" or "// Comment" instead
    |       |     (Squiz.Commenting.InlineComment.DocBlock)
 78 | ERROR | [x] The close comment tag must be the only content on the line (Generic.Commenting.DocComment.ContentBeforeClose)
--------------------------------------------------------------------------------------------------------------------------------
```

Reply:
I see... Then it should be ok as it is, even it's not perfect 😞